api/model/organization/OrganizationData.v1.yaml

```diff
@@ -0,0 +1,51 @@
+title: OrganizationData
+type: object
+description: Data of an organization.
+properties:
+  name:
+    type: string
+    example: Unipd Math
+  description:
+    type: string
+    example: Dipartimento di Matematica di Unipd
+  placeIds:
+    type: array
+    items:
+      $ref: ../GenericId.v1.yaml
+  organizationType:
+    type: string
+    enum:
+      - private
+      - public
+  ldapConfiguration:
+    description: The information required to allow the configuration of LDAP authentication for private organizations.
+    type: object
+    properties:
+      url:
+        type: string
+        format: uri
+        description: "The organization's LDAP server URL."
+      searchQuery:
+        type: string
+        description: An LDAP search query that returns the list of users registered to it.
```

Comment:
In "registered to it" a cosa si riferisce "it"?

Reply:
specifico "to the LDAP server"